lib/absinthe/schema/notation/sdl_render.ex

```diff
@@ -118,7 +118,7 @@ defmodule Absinthe.Schema.Notation.SDL.Render do
       "type",
       concat([
         string(object_type.name),
-        implements(object_type.interface_blueprints)
```

Comment:
as far as I could tell, `interface_blueprints` was always empty, where as `interfaces` includes a list of names of implemented interfaces, which is really all we need to render the SDL.

Reply:
should we find out why that was empty?  it might be what we need to get the correct name...